VirtoCommerce.Storefront.Model/Catalog/ProductSearchCriteria.cs

```diff
@@ -62,11 +62,10 @@ public override object Clone()
         }
 
         private void Parse(NameValueCollection queryString)
-        {
+        {        
+            IsFuzzySearch = queryString.Get("fuzzy").EqualsInvariant(bool.FalseString);
```

Comment:
Why comparing with False? Thus we set IsFuzzySearch=true only when passing fuzzy=false.